services/web/server/src/simcore_service_webserver/users/_preferences_db.py

```diff
@@ -0,0 +1,56 @@
+from aiohttp import web
+from models_library.products import ProductName
+from models_library.user_preferences import AnyBaseUserPreference, PreferenceName
+from models_library.users import UserID
+from simcore_postgres_database.utils_user_preferences import UserPreferencesRepo
+
+from ..db.plugin import get_database_engine
+
+
+def _get_user_preference_name(user_id: UserID, preference_name: PreferenceName) -> str:
```

Comment:
I find all this encoding difficult to follow. we have all these info in the db. the db can tell you if you are setting non unique stuff there.